src/diffusers/video_processor.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+
+import numpy as np
+import PIL
+import torch
+
+from .image_processor import VaeImageProcessor
+
+
+class VideoProcessor:
+    """Simple video processor."""
+
+    @staticmethod
+    def tensor2vid(video: torch.Tensor, processor: "VaeImageProcessor", output_type: str = "np"):
```

Comment:
can you have the methods like frames2gif or frames2mpeg in here as well

Reply:
#5881

Reply:
I think we can add this functionality to the export functions we have. Generally the pipelines always return an np array, a torch tensor or PIL image/ PIL image list.

Reply:
I don't think so. Here's an example where `tensor2vid` is being used when returning the output:

https://github.com/huggingface/diffusers/blob/49b959b5408b97274e2ee423059d9239445aea26/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L565